core/trino-parser/src/main/java/io/trino/sql/QueryUtil.java

```diff
@@ -162,7 +162,7 @@ public static Relation aliased(Relation relation, String alias, List<String> col
                 relation,
                 identifier(alias),
                 columnAliases.stream()
-                        .map(QueryUtil::identifier)
+                        .map(QueryUtil::quotedIdentifier)
```

Comment:
There are other sites than `describe`-rewrites involving a `simpleQuery` over `aliased`. They also need to be updated so that the selected column names are quoted.
These are:
- methods `visitShowSession()` and `visitShowFunctions()` in `ShowQueriesRewrite`
- method `rewriteShowStats` in `ShowStatsRewrite`

Reply:
ok. I'll fix them and add test case for them.